ios/browser/api/url/url_utils.mm

```diff
@@ -95,6 +96,27 @@ - (bool)brave_hasScheme:(NSString*)scheme {
   return net::GURLWithNSURL(self).SchemeIs(
       base::SysNSStringToUTF8([scheme lowercaseString]));
 }
+
+- (nullable NSURL*)
+    brave_stripTrackerParamsUsingInitiatorURL:(NSURL*)initiatorURL
+                            redirectSourceURL:(NSURL*)redirectSourceURL
+                                requestMethod:(NSString*)requestMethod
+                           isInternalRedirect:(BOOL)isInternalRedirect {
+  GURL initiator_url = net::GURLWithNSURL(initiatorURL);
+  GURL redirect_source_url = net::GURLWithNSURL(redirectSourceURL);
+  GURL request_url = net::GURLWithNSURL(self);
+  const auto request_method = base::SysNSStringToUTF8(requestMethod);
+
+  auto filtered_url = query_filter::MaybeApplyQueryStringFilter(
+      initiator_url, redirect_source_url, request_url, request_method,
+      isInternalRedirect);
```

Comment:
you can omit this with https://github.com/brave/brave-core/pull/19633/files#r1290594995
but even without that change you could have just left it out of the obj-c interface and hardcoded false here

Reply:
like above, decided to keep this variable as its useful in iOS.